types/es-abstract/scripts/intrinsics-data.ts

```diff
@@ -0,0 +1,511 @@
+    '%Promise_resolve%': typeof Promise === 'undefined' ? undefined : Promise.resolve,
+    '%Proxy%': typeof Proxy === 'undefined' ? undefined : Proxy,
+    '%RangeError%': RangeError,
+    '%RangeErrorPrototype%': RangeError.prototype,
+    '%ReferenceError%': ReferenceError,
+    '%ReferenceErrorPrototype%': ReferenceError.prototype,
+    '%Reflect%': typeof Reflect === 'undefined' ? undefined : Reflect,
+    '%RegExp%': RegExp,
+    '%RegExpPrototype%': RegExp.prototype,
+    '%Set%': typeof Set === 'undefined' ? undefined : Set,
+    '%SetIteratorPrototype%': typeof Set === 'undefined' || !hasSymbols ? undefined : getProto(new Set()[Symbol.iterator]()),
+    '%SetPrototype%': typeof Set === 'undefined' ? undefined : Set.prototype,
+    '%SharedArrayBuffer%': typeof SharedArrayBuffer === 'undefined' ? undefined : SharedArrayBuffer,
+    '%SharedArrayBufferPrototype%': typeof SharedArrayBuffer === 'undefined' ? undefined : SharedArrayBuffer.prototype,
+    '%String%': String,
+    '%StringIteratorPrototype%': hasSymbols ? getProto(''[Symbol.iterator]()) : undefined,
+    '%StringPrototype%': String.prototype,
+    '%Symbol%': hasSymbols ? Symbol : undefined,
+    '%SymbolPrototype%': hasSymbols ? Symbol.prototype : undefined,
+    '%SyntaxError%': SyntaxError,
+    '%SyntaxErrorPrototype%': SyntaxError.prototype,
+    '%ThrowTypeError%': ThrowTypeError,
+    '%TypedArray%': TypedArray,
+    '%TypedArrayPrototype%': TypedArray ? TypedArray.prototype : undefined,
+    '%TypeError%': TypeError,
+    '%TypeErrorPrototype%': TypeError.prototype,
+    '%Uint8Array%': typeof Uint8Array === 'undefined' ? undefined : Uint8Array,
+    '%Uint8ArrayPrototype%': typeof Uint8Array === 'undefined' ? undefined : Uint8Array.prototype,
+    '%Uint8ClampedArray%': typeof Uint8ClampedArray === 'undefined' ? undefined : Uint8ClampedArray,
+    '%Uint8ClampedArrayPrototype%': typeof Uint8ClampedArray === 'undefined' ? undefined : Uint8ClampedArray.prototype,
+    '%Uint16Array%': typeof Uint16Array === 'undefined' ? undefined : Uint16Array,
+    '%Uint16ArrayPrototype%': typeof Uint16Array === 'undefined' ? undefined : Uint16Array.prototype,
+    '%Uint32Array%': typeof Uint32Array === 'undefined' ? undefined : Uint32Array,
+    '%Uint32ArrayPrototype%': typeof Uint32Array === 'undefined' ? undefined : Uint32Array.prototype,
+    '%URIError%': URIError,
+    '%URIErrorPrototype%': URIError.prototype,
+    '%WeakMap%': typeof WeakMap === 'undefined' ? undefined : WeakMap,
+    '%WeakMapPrototype%': typeof WeakMap === 'undefined' ? undefined : WeakMap.prototype,
+    '%WeakSet%': typeof WeakSet === 'undefined' ? undefined : WeakSet,
+    '%WeakSetPrototype%': typeof WeakSet === 'undefined' ? undefined : WeakSet.prototype,
+};
+setProto(BASE_INTRINSICS, null);
+
+// prettier-ignore
+export const LEGACY_ALIASES: { [intrinsic: string]: string[] } = {
+	'%ArrayBufferPrototype%': ['ArrayBuffer', 'prototype'],
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/es-abstract/scripts/intrinsics-data.ts;
  Line: 168;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.